github/event_types.go

```diff
@@ -106,3 +106,15 @@ type IssueCommentEvent struct {
 	Repo   *Repository `json:"repository,omitempty"`
 	Sender *User       `json:"sender,omitempty"`
 }
+
+// CommitCommentEvent represents the payload delivered by CommitComment webhook.
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#commitcommentevent
+type CommitCommentEvent struct {
+	Comment *RepositoryComment `json:"comment,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	Action *string     `json:"action,omitempty"`
```

Comment:
ah, you're right.  I was looking at the wrong thing.  Just tried leaving a comment on a commit to confirm (https://api.github.com/repos/google/go-github/events):

```json
[
  {
    "id": "3652449134",
    "type": "CommitCommentEvent",
    "actor": {
      "id": 1112,
      "login": "willnorris",
      "gravatar_id": "",
      "url": "https://api.github.com/users/willnorris",
      "avatar_url": "https://avatars.githubusercontent.com/u/1112?"
    },
    "repo": {
      "id": 10270722,
      "name": "google/go-github",
      "url": "https://api.github.com/repos/google/go-github"
    },
    "payload": {
      "comment": {
        "url": "https://api.github.com/repos/google/go-github/comments/16134778",
        "html_url": "https://github.com/google/go-github/commit/1ef1ae6f862a1e829a33c09b5a76673e249e5c88#commitcomment-16134778",
        "id": 16134778,
        "user": {
          "login": "willnorris",
          "id": 1112,
          "avatar_url": "https://avatars.githubusercontent.com/u/1112?v=3",
          "gravatar_id": "",
          "url": "https://api.github.com/users/willnorris",
          "html_url": "https://github.com/willnorris",
          "followers_url": "https://api.github.com/users/willnorris/followers",
          "following_url": "https://api.github.com/users/willnorris/following{/other_user}",
          "gists_url": "https://api.github.com/users/willnorris/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/willnorris/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/willnorris/subscriptions",
          "organizations_url": "https://api.github.com/users/willnorris/orgs",
          "repos_url": "https://api.github.com/users/willnorris/repos",
          "events_url": "https://api.github.com/users/willnorris/events{/privacy}",
          "received_events_url": "https://api.github.com/users/willnorris/received_events",
          "type": "User",
          "site_admin": false
        },
        "position": 4,
        "line": 27,
        "path": "github/github.go",
        "commit_id": "1ef1ae6f862a1e829a33c09b5a76673e249e5c88",
        "created_at": "2016-02-17T00:30:00Z",
        "updated_at": "2016-02-17T00:30:00Z",
        "body": "leaving a comment to try and force a `CommitCommentEvent`"
      }
    },
    "public": true,
    "created_at": "2016-02-17T00:30:00Z",
    "org": {
      "id": 1342004,
      "login": "google",
      "gravatar_id": "",
      "url": "https://api.github.com/orgs/google",
      "avatar_url": "https://avatars.githubusercontent.com/u/1342004?"
    }
  }
]
```